languages/elixir/exercises/concept/chessboard/.docs/instructions.md

```diff
@@ -0,0 +1,39 @@
+As a chess enthusiast, you would like to write your own version of the game. Yes, there maybe plenty of implementations of chess available online already, but yours will be unique!
+
+But before you can let your imagination run wild, you need to take care of the basics. Let's start by generating the board.
+
+Each square of the chessboard is identified by a letter-number pair. The vertical columns of squares, called files, are labeled A through H. The horizontal rows of squares, called ranks, are numbered 1 to 8.
+
+## 1. Define the rank range
+
+Implement the `rank_range/0` function. It should return a range of integers, from 1 to 8.
+
+```elixir
+Chessboard.rank_range()
+```
+
+## 2. Define the file range
+
+Implement the `file_range/0` function. It should return a range of integers, from the code point of the uppercase letter A, to the codepoint of the uppercase letter H.
```

Comment:
I have noticed that between different documents, we use both the spelling "code point" and "codepoint". I'm not sure which one is correct.

Reply:
I don't have a strong preference, so I have probably been careless with this.  [Wikipedia's](https://en.wikipedia.org/wiki/Code_point) entry is **code point** so probably should change things to be consistent with that.

Reply:
Elixir's documentation seems to agree that it's spelled *code point*, yet the function is called `String.codepoints`, not `String.code_points` 🤔 https://hexdocs.pm/elixir/String.html#codepoints/1